README.md

```diff
@@ -1,2 +1,29 @@
 # ApprenticeProgress
-A platform to track Ben's progress through his thoughtbot apprenticeship 
+A platform to track Ben's progress through his thoughtbot apprenticeship.
+
+# Goals for tracking progress
+These are a combination of [Makers](https://makers.tech/) learning criteria for their Academy course and a summary of thoughtbot's expectations for Developers.
+
+These are the most important statements a thoughtbot apprenticeship should be able to state. Evidencing them all by the end of the apprenticeship should be the _main_ evidence required for my promotion to Developer.
+
+*I'm team thoughtbot*
+* I contribute to thoughtbot being a good place to work
+* I iterate over my existing knowledge effectively
+* I’m a great pair programmer
+* I help others become better
+* I’m passionate about software/products
+* I set my own direction
+
+*I'm a developer*
+* I can write high quality code for React Native (OOP, easy to understand, easy to change
+* I can write high quality test suites for React Native
+* I can build apps in React Native
+* I understand how React Native works
+* I understand how the web works
+* I understand how databases work
+
+*I'm a consultant*
+* I communicate clearly with clients
+* I can clearly explain to others where/when thoughtbot can add value
+* I understand how technical/design decisions will impact clients
+* My teams build software clients’ love
```

Comment:
Would it be helpful to [make these checklists]?

[make these checklists]: https://blog.github.com/2014-04-28-task-lists-in-all-markdown-documents/

Reply:
@purinkle - Can do though I think the real progress will be tracked in the app when built. Maybe as a separate document I'll do an initial assessment - like you were asking me for earlier.